src/user/views/editor_views.py

```diff
@@ -17,15 +17,21 @@
 from hub.models import Hub
 
 
-def resolve_timeframe_for_contribution(startDate, endDate):
+def resolve_timeframe_for_contribution(startDate, endDate, query_key=None):
```

Comment:
camelCase -> snake_case

Reply:
I'll change it. I think it was updated by pat.